migrations/versions/54e57fb2e1c6_add_contract_source_to_records_and_items.py

```diff
@@ -0,0 +1,26 @@
+"""add contract source to records and items
+
+Revision ID: 54e57fb2e1c6
+Revises: e46b30dd3ff5
+Create Date: 2024-06-26 15:04:24.092549
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '54e57fb2e1c6'
+down_revision = 'e46b30dd3ff5'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('records', sa.Column('contract_source', sa.ARRAY, index=True))
+    op.add_column('items', sa.Column('contract_source', sa.ARRAY, index=True))
```

Comment:
Are we pretty positive that this will be a list of strings and we won't need to store any additional information besides the name of the contract source? 

Also can this be nullable? We may have to specify in the nullable param. 

https://alembic.sqlalchemy.org/en/latest/ops.html

Reply:
If I add the code `sa.String` as a parameter to the `sa.ARRAY` like `sa.ARRAY(sa.String)`, then the list_type (the parameter that is being filled in for `sa.ARRAY`) will be guaranteed to be string values. Also, adding a column like in the other database migrations will be nullable by default because the nullable parameter is only used when altering a table or creating a table which the records and items table already allow null values.